exercises/practice/high-scores/HighScoresTest.php

```diff
@@ -33,55 +12,117 @@ public static function setUpBeforeClass(): void
         require_once 'HighScores.php';
     }
 
+    /*
```

Comment:
Please make all of these UUID comments to DocBlocks like this:

```suggestion
    /**
```